.travis.yml

```diff
@@ -23,7 +23,7 @@ install:
   # converting \n to \r\n.
   - "git config --global core.autocrlf input"
   - "git checkout -- ."
-  - "go install ./wire/cmd/wire"
+  - "go install github.com/google/wire/cmd/wire"
```

Comment:
Are we generating code in the tests though?

Reply:
We're running `wire check` to ensure that there are no errors in the provider sets we give. See here: https://github.com/google/go-cloud/blob/87e273223b452e987ff654f016887ab80a6f9252/internal/testing/runchecks.sh#L38